controllers/Dockerfile

```diff
@@ -12,8 +12,6 @@ RUN --mount=type=cache,target=/go/pkg/mod \
 
 COPY model model
 COPY controllers controllers
-COPY kpack-image-builder kpack-image-builder
-COPY job-task-runner job-task-runner
 COPY statefulset-runner statefulset-runner
```

Comment:
If you are splitting the `statefulset-runner` into its own deployment with its own docker image this line should go.

Currently if it is removed there are compilation errors, because `controllers` refer to a [constant](https://github.com/cloudfoundry/korifi/blob/86a982af259f8a6411453f5976824dc6a76d36f4/statefulset-runner/controllers/appworkload_controller.go#L54) defined by `statefulset-runner`. However this constant is a duplicate so we car simply refer to the [one in the korifi types](https://github.com/cloudfoundry/korifi/blob/86a982af259f8a6411453f5976824dc6a76d36f4/controllers/api/v1alpha1/shared_types.go#L10). With this change we should be able to get rid of `statefulset-runner` in the `controllers` Dockerfile.